### PR TITLE
feat: show V4 API events

### PR DESCRIPTION
**Pull request by jgiovaresco**
## Issue

https://gravitee.atlassian.net/browse/APIM-1423

## Description

Display API Events of V4 APIs.

As described in this [comment](https://gravitee.atlassian.net/browse/APIM-1423?focusedCommentId=24495), we show events and audits in 2 distinct tables.
Also, there was no design so I've reused what we already show for V2 APIs. However, I've implemented pagination in the SearchApiEvents endpoint. Therefore, this screen will only show the ten latest events of the API.

image

## Additional context

I didn't put event's `updatedAt` in the OpenAPI definition because an event should be immutable and therefore the `updatedAt` will always be equal to `createdAt`. I can revise that if someone doesn't like that 🙈 

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-vggutfgdxo.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/api-traffic-v4/audit-logs/components/api-events-table/api-events-table.component.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  public set events(value: Event[] | undefined) {
+    this._events = value;
+    // this._events = value.map((event) => ({
+    //   type: event.type,
+    //   createdAt: event.createdAt,
+    //   user: {
+    //     id: event.initiator.id,
+    //     displayName: event.initiator.displayName,
+    //   },
+    // }));
```
Comment (Okhelifi) [inline gravitee-apim-console-webui/src/management/api/api-traffic-v4/audit-logs/components/api-events-table/api-events-table.component.ts]: To remove ?

File: gravitee-apim-console-webui/src/management/api/api-traffic-v4/audit-logs/components/api-events-table/api-events-table.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import { Event } from '../../../../../../entities/management-api-v2/event';
+
+// type EventsTableVM = {
+//   type: string;
+//   createdAt: Date;
+//   user: {
+//     id: string;
+//     displayName: string;
+//   };
+// };
```
Comment (Okhelifi) [inline gravitee-apim-console-webui/src/management/api/api-traffic-v4/audit-logs/components/api-events-table/api-events-table.component.ts]: To remove too ?

File: gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml [hunk 3]
```diff
@@ -5755,6 +5816,15 @@ components:
...
                     type: string
+        eventTypes:
+            name: types
+            description: List of event types to filter on.
+            in: query
+            explode: false
+            schema:
+                type: array
+                items:
+                    type: string
```
Comment (Okhelifi) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml:5846]: Not a blocker as the string is converted into an enum in the buildInput method but should we have a list of enum ?

**Review by Okhelifi — COMMENTED** [review]
Some minor comments. Like discussed together if you could move the "Audit logs" tag in the configuration menu entry it would help me 😅

File: gravitee-apim-console-webui/src/management/api/api-traffic-v4/audit-logs/components/api-events-table/api-events-table.component.ts (hunk 1 above)
Comment (jgiovaresco) [reply gravitee-apim-console-webui/src/management/api/api-traffic-v4/audit-logs/components/api-events-table/api-events-table.component.ts]: oups 🙈

**Review by Okhelifi — APPROVED** [review]
Tested locally all good for me. Good job 👍🏼

**Pull request merged**